### PR TITLE
Psuedo random scenery [hackathon]

### PR DESCRIPTION
**Pull request by DefinitelyMaybe**
Hi all,

This is my submission for the hackathon.

Enjoy!

### CODE REVIEW

File: apps/docs/src/examples/geometry/psuedo-random-scenery/adjusted-sampling/poissonDiscSampling.ts
Comment (ixxie) [inline apps/docs/src/examples/geometry/psuedo-random-scenery/adjusted-sampling/poissonDiscSampling.ts:1]: Rename to `sampling.ts`.

**Review by ixxie — CHANGES_REQUESTED** [review]
In addition to the other comments, I would like you to consider:

- [ ] Refactoring `Scene.svelte` files to reduce filesize: the scenes have become quite large, making it a bit hard to follow the file. Could you try and break some of the code out into separate files? I'm thinking the standard lighting / camera stuff can stay in Scene, and we seperate the rest to a `Pseudorandom.svelte` file.
- [ ] Rename examples to streamline them and fit the new text - perhaps something like `uniform-pseudorandom-scenery`, `poisson-psuedorandom-scenery` and `variable-poisson-pseudorandom-scenery`.

**Comment by DefinitelyMaybe** [comment]
All very good suggestions! thank you!

**Review by ixxie — APPROVED** [review]

**Comment by ixxie** [comment]
Looks Great to Me! Good work @DefinitelyMaybe!
